package.json

```diff
@@ -67,6 +67,7 @@
   },
   "dependencies": {
     "cids": "^1.0.0",
+    "interface-ipld-format": "0.0.2",
```

Comment:
where's the source of this? https://github.com/ipld/interface-ipld-format doesn't match https://unpkg.com/browse/interface-ipld-format like the package.json suggests it should.

Reply:
You found it, but for others it's here: https://github.com/ipld/interface-ipld-format/pull/56

We should merge that and `v1` it before merging this.